main.go

```diff
@@ -167,12 +167,7 @@ func main() {
 		OnInit: func(p *plugin.Plugin) {
 			// read config -- cant get from node on plugin initialization step
 			network, _ = p.Args.String("trustedcoin-network")
-			if network != "" {
-				p.Log("network type: " + network)
-			} else {
-				network = "main"
-				p.Log("default network type: " + network)
-			}
```

Comment:
Oh, sorry, I hadn't seen this default mechanism here.

Reply:
No. It is bad. Thank you for the right suggestion. I will test changes and write back then.